components/more-less/more-less.js

```diff
@@ -1,4 +1,5 @@
 import '../button/button-subtle.js';
+import '../expand-collapse/expand-collapse-content.js';
```

Comment:
Looks like this import statement is left over from when you were trying to use `d2l-expand-collapse`.

Reply:
Change has been made, thanks!